frontend/app/src/App.tsx

```diff
@@ -13,6 +13,7 @@ import { initCfmm } from './contracts/cfmm';
 import { logger } from './utils/logger';
 import { getNodePort, getNodeURL } from './utils/settingUtils';
 import ModalContainer from './components/modals/ModalContainer';
+import theme from './components/theme/theme';
```

Comment:
theme should not within components directory, it should have a directory on it's own,